docs/acceptance/db.md

```diff
@@ -0,0 +1,65 @@
+## Database Deliverable Acceptance Test 
+
+### Description of Deliverable
+
+This deliverable includes the implementation of an ETL process, the design of a database schema, the selection of a format for raw data storage, infrastructure as code, database configuration, and accompanying documentation.
+
+The acceptance test below provides steps to verify that the deliverable meets our agreed-upon specifications.
+
+### Input Data
+
+The input data is stored in Parquet format on AWS S3 (object storage), specifically in the file `space2stats_updated.parquet`. Any additional fields must be appended to this file. The Parquet file is tabular with the following columns:
+- `hex_id`
+- `{aggregation_method[sum, mean, etc.]}_{variable_name}_{year}`
+
+### Database Setup
+
+You can use a local database for this acceptance test by running the following command in the root directory:
+
+```bash
+docker-compose up
+```
+
+Alternatively, you can connect to a remote database, such as the Tembo database used for production.
+
+### Data Ingestion
+
+Set the database environment variables in `db.env`:
+
+```bash
+DB_HOST=localhost
```

Comment:
It's really handy to just use the standard Postgres environment variables rather than more generic DB_ variables. This allows you to just set the environment and run the app or use psql or any other tooling that uses the standard environment variables without having to set the variables multiple times or to need to use command line parameters. PGPORT, PGDATABASE, PGUSER, PGPASSWORD

Reply:
Opened a PR for this specific change.